jwst/pipeline/calwebb_spec3.py

```diff
@@ -61,12 +61,13 @@ def process(self, input):
             The exposure or association of exposures to process
         """
         self.log.info('Starting calwebb_spec3 ...')
+        self.allowed_exptypes = ['science','background']

```

Comment:
This can probably be `allowed_exptypes` instead of `self.allowed_exptypes` as it is not being used outside of `process()` method.  Unless you'd like to make this attribute available on the `ModelContainer` itself for introspection, which I suppose could be useful.